src/wp-settings.php

```diff
@@ -114,6 +114,7 @@
 require ABSPATH . WPINC . '/class-wp.php';
 require ABSPATH . WPINC . '/class-wp-error.php';
 require ABSPATH . WPINC . '/pomo/mo.php';
+require ABSPATH . WPINC . '/database-types.php';
```

Comment:
Typo? The `[tests/phpunit/tests/database-types.php](https://github.com/WordPress/wordpress-develop/pull/3124/files#diff-3b12799a02b64077d350d257ceafa3eb9e7924602825d785930eb5a95328a2ec)` file is included automatically when running PHPUnit.